jenkins/image_building/initrd_sdk/unlock-mount-luks.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```

Comment:
This file is clearly coming from somewhere, can you document the source/license/copyright etc?

Reply:
I have wrote this myself completely. This file does not exist anywhere else AFAIK.